generators/app/index.js

```diff
@@ -61,6 +61,12 @@ module.exports = class extends Generator {
         default: 'hwWxD5cB6LtaCB0GOcbaxiOI2eaFoC4rIT9jh51DCdB6p9IZrHTMRuFUM72xIjm',
         when: (answers) => answers.auth === 'JWT',
       },
+      {
+        type: 'list',
+        name: 'packageManager',
+        message: 'which package manager do you wish to use?',
```

Comment:
semantic, but "which package manager do you want to use?" over "which package manager do you wish to use?"